config/environments/production.rb

```diff
@@ -16,12 +16,12 @@
     authentication:       :plain,
     user_name:            ENV["SENDGRID_USERNAME"],
     password:             ENV["SENDGRID_PASSWORD"],
-    domain:               'heroku.com',
+    domain:               'ec2-52-88-40-177.us-west-2.compute.amazonaws.com',
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.